src/app/user/register/index.njk

```diff
@@ -7,14 +7,12 @@
 {% endblock %}
 
 {% block header %}
-{% include "header-pre-user.njk" %}
+  {% include "header-pre-user.njk" %}
```

Comment:
Not convinced there needs to be a pre-user and user header. Will need to refactor in due time.